enclave_wrangler/objects_api.py

```diff
@@ -892,16 +941,20 @@ def cli():
     #     del kwargs_dict['download_favorite_objects']
     #     run(**kwargs_dict)
 
+    # New args added from 9/12/2023
+    parser.add_argument(
+        '-A', '--find-and-add-missing-csets-to-db', action='store_true',
```

Comment:
### CLI
Reactivated `objects_api` CLI w/ 2 commands:
- `--find-and-add-missing-csets-to-db`
- `--find-missing-csets`

I updated the DB counts, then ran `--find-and-add-missing-csets-to-db`, and then ran the counts again.

We can use this command in the future if we need to do this again.